static/resources/math3d_app.js

```diff
@@ -266,14 +271,18 @@ app.controller('addObjectCtrl',['$scope', '$sce', function($scope, $sce) {
 
     $scope.createNewObject = function(type){
         var metaMathObj = {type:type, settings:{}};
-        var mathObj = MathObject.renderNewObject(math3d, metaMathObj);
+
+        if (math3d.selected !== undefined) {
+            var insertionPoint = math3d.selected.getMathtreePosition();
+        }
+        var mathObj = MathObject.renderNewObject(math3d, metaMathObj, insertionPoint);
```

Comment:
@stardust66 I think that when user creates a new object, it should automatically be selected. Do you foresee any disadvantages?

I realize you probably haven't thought about this PR in a while ... I'm sorry!

Reply:
Yeah I agree. Doing that now.